plugins/extractors/elastic/elastic.go

```diff
@@ -101,22 +102,32 @@ func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) (err error)
 		return
 	}
 	x := reflect.ValueOf(r["indices"]).MapRange()
+	var indexes []string
 	for x.Next() {
 		indexName := x.Key().String()
-		docProperties, err1 := e.listIndexInfo(x.Key().String())
+		indexes = append(indexes, indexName)
+	}
+	sort.Strings(indexes)
+	for _, index := range indexes {
```

Comment:
let's rename `index` to `indexName`, `index` can mean another thing in a for loop.